modules/FormModel.js

```diff
@@ -322,40 +495,111 @@
     checkIfModelsAreStale: function() {
       var staleModels = {},
         currentHashValues = this.__generateAllHashValues();
-      _.each(this.__getAllModels(true), function(model) {
+      _.each(this.getTrackedModels(), function(model) {
         this.isModelStale(model, staleModels, currentHashValues);
       }, this);
       return _.values(staleModels);
     },
 
+    /************** Private methods **************/
+
     /**
-     * Sets up a listener to update the form model if the model's field changes.
+     * Sets up a listener to update the form model if the model's field (or any field) changes.
      * @param model {Backbone.Model} the object model from which this form model will start listen to changes
-     * @param field {String} the field name that it will start listening to.
-     * @method listenToModelField
+     * @param [field] {String} the field name that it will start listening to. If no field is given, it will listen to the general 'change' event.
+     * @method __listenToModelField
+     * @private
      */
-    listenToModelField: function(model, field) {
-      var eventName = 'change:' + field;
-      this.listenTo(model, eventName, _.bind(this.__updateFormField,
-          {formModel: this, field: field}));
+    __listenToModelField: function(model, field) {
+      var eventName = field ? ('change:' + field) : 'change';
```

Comment:
Just writing it to see how it looks... hoisting is annoying:
```
field = field ? ':' + field : '';
var eventName = 'change' + field;
```

what you have already is probably clearer.